my_class.rb

```diff
@@ -0,0 +1,12 @@
+require 'MyEnumerable'
```

Comment:
- [ ] Use `require_relative` to import `MyEnumerable` since using `require_relative` is for files in your directory instead of `require` . Require should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."